swim/handlers.go

```diff
@@ -56,18 +61,19 @@ func notImplementedHandler(ctx json.Context, req *emptyArg) (*emptyArg, error) {
 
 func (n *Node) registerHandlers() error {
 	handlers := map[string]interface{}{
-		"/protocol/join":      n.joinHandler,
-		"/protocol/ping":      n.pingHandler,
-		"/protocol/ping-req":  n.pingRequestHandler,
-		"/admin/debugSet":     notImplementedHandler,
-		"/admin/debugClear":   notImplementedHandler,
-		"/admin/gossip":       n.gossipHandler, // Deprecated
-		"/admin/gossip/start": n.gossipHandlerStart,
-		"/admin/gossip/stop":  n.gossipHandlerStop,
-		"/admin/tick":         n.tickHandler, // Deprecated
-		"/admin/gossip/tick":  n.tickHandler,
-		"/admin/member/leave": n.adminLeaveHandler,
-		"/admin/member/join":  n.adminJoinHandler,
+		"/protocol/join":             n.joinHandler,
+		"/protocol/ping":             n.pingHandler,
+		"/protocol/ping-req":         n.pingRequestHandler,
+		"/admin/debugSet":            notImplementedHandler,
+		"/admin/debugClear":          notImplementedHandler,
+		"/admin/gossip":              n.gossipHandler, // Deprecated
+		"/admin/gossip/start":        n.gossipHandlerStart,
+		"/admin/gossip/stop":         n.gossipHandlerStop,
+		"/admin/healpartition/disco": n.partitionHealerHandler,
```

Comment:
We might implement different ways of healing. With faulties or by specifying a host we want to heal with. I don't think it's a big deal that the user has to call it like this.